app/assets/stylesheets/spotlight/_accessibility.scss

```
@@ -2,7 +2,7 @@
 
 // Increase visibility of Firefox focus indicator
 @-moz-document url-prefix() {
-  &:focus {
+  #{if(&, "&", "*")}:focus {
```

```suggestion
  :focus {
```

Looking at the output CSS, at least, the  `&` doesn't seem to be doing anything for us anyway (and we're not in a mixin context where we need dynamic binding or anything)